### ISSUE TITLE
"eMylo Wifi Switch 2 Gang" connection ok, but no datapoint found error

### ISSUE THREAD

**Issue by flyingdisk** [rospogrigio/localtuya#1070]
Hi, 

i'm trying to configure eMylo Wifi Switch 2 Gang with localtuya

the connection is ok, but integration say "no datapoint found"

- Localtuya version: 4.1.0  

- Home Assistant Core version:  2022.10.1  
- [x] Are you using the Home Assistant Tuya Cloud component ? 
- [x] Are you using the Tuya App in parallel ? 

## Steps to reproduce

1. add new device
2. select new device
3. add all infos
4. localtuya try to add device and reply "connection ok but datapoint not found"

## Provide Home Assistant taceback/logs
```txt
[DebugLog_tuyaHA.txt](https://github.com/rospogrigio/localtuya/files/9734959/DebugLog_tuyaHA.txt)

```

**Comment by Noneofinterest** [comment]
Same here. I red a 2020 post describing the same problem. Their solution was to add Local Tuya settings to config.yaml, but a better solution would be very appreciated.

**Comment by robertrebor** [comment]
same problem is for me. I tried add 2 gang switch and I have following message: Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists.

I am new user in github, what information I need to provide for find a solution?
